src/stories/SearchBar/index.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import {storiesOf} from '@storybook/react';
+import {withInfo} from '@storybook/addon-info';
+import {Button} from 'react-bootstrap';
+import {Icon} from '../../indigo/components';
+import {action} from '@storybook/addon-actions';
+
+import SearchBar from '../../indigo/components/SearchBar';
+
+
+const demoContent = (
+  <Button bsStyle="default">
+    <Icon.Help className={'icon-size-20'}/> Demo Button
+  </Button>
+)
+storiesOf('SearchBar', module)
+  .add(
+    'Basic usage',
+    withInfo({
+      text: `
+        usage: ex, tr, wr, orch, versions
+        `,
+      inline: true,
+    })(() => {
+      return (
+        <SearchBar
+          query=""
+        />
+      );
+    })
+  )
+  .add(
+    'Searchbar with actions',
+    withInfo({
+      text: `
+        usage: config rows, config detail
+        `,
+      inline: true,
+    })(() => {
+      return (
+        <SearchBar
+          query=""
+          actions={demoContent}
+        />
+      );
+    })
+  )
+  
+  .add(
+    'Custom placeholder',
+    withInfo({
+      text: `
+        usage: ex, tr, wr, orch, versions
+        `,
+      inline: true,
+    })(() => {
+      return (
+        <SearchBar
+          query=""
+          placeholder="Search by component name, desription or id"
+        />
+      );
+    })
+  )
+  .add(
+    'Events demo',
+    withInfo({
+      text: `
+        onChange & onSubmit demo
+        `,
+      inline: true,
+    })(() => {
+      return (
+        <SearchBar
+          query=""
+          onChange={action('changed content')}
```

Comment:
@janmichek za toto :+1: ... pre ten `onChange` to ale nefici, vid moj komentar.

Reply:
fixed